app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -18,16 +19,19 @@ class Api::V2::ContentViewVersionsController < Api::V2::ApiController
     param :composite_version_id, :identifier, :desc => N_("Filter versions that are components in the specified composite version"), :required => false
     param_group :search, Api::V2::ApiController
     def index
+      includes = [:content_view, :environments, :composite_content_views, :history => :task]
+      sort = "#{versions.table_name}.major desc, #{versions.table_name}.minor desc"
```

Comment:
I think this should be `ContentViewVersion.table_name` ? Or, `resource_class.table_name`